docs/process/release.md

```diff
@@ -39,6 +39,27 @@ Say the following PR is merged;
 
 Only the first 2 commits should be cherry picked to the patch branch, as the last 3 are related to a new feature.
 
+Cherry picking is done by checking out the patch branch and cherry picking the new commit onto it.
+This should be done after merging to master.
+
+```shell
+# Make sure commit to upstream/master is present local repo.
+git fetch upstream master
+
+# Check out patch branch from upstream.
+git fetch upstream 9.0.x
```

Comment:
consider making the command more generic, like `git fetch upstream <patch branch>`

Reply:
Done. Added a line at the top to point to what naming convention to look for to identify the patch branch.